app/views/content/non-uk-teachers/non-uk-qualifications.md

```diff
@@ -0,0 +1,133 @@
+
+## Get help checking your qualifications
+
+For help with qualifications, you can use the live chat service at the bottom of this page, or call us on +44 800 389 2500. 
+
+We're available Monday to Friday between 8:30am and 5:30pm (UK local time) except on [bank holidays](https://www.gov.uk/bank-holidays) in England. Phone calls will be charged at your standard rate.
+
+When you call or chat, we can tell you if you are eligible for a [teacher training adviser](/teacher-training-advisers). 
+
+
+## Qualifications you'll need to train to teach
+
+To train to teach in England, you'll need:
+
+* a degree which meets the standard of a UK bachelor’s degree (this can come from a non-UK university)
+* an English language qualification which meets the standard of a grade 4 General Certificate of Secondary Education (GCSE) – a qualification awarded for exams taken at age 16 in England
+* a maths qualification which meets the standard of a grade 4 GCSE
+* if teaching children aged 3 to 11 in primary school, a physics, chemistry or biology (science) qualification which meets the standard of a grade 4 GCSE
+
+You'll also need to:
+
+* complete a fitness questionnaire before starting training
+* undergo safeguarding and criminal record checks carried out by your training provider or employer
+
+## English language, maths and science proficiency
+
+If you’ve reached a certain proficiency in an English language test, you can use this as evidence you meet the standard of a grade 4 GCSE in English – for example: 
+
+* [International English Language Testing System (IELTS)](https://www.ielts.org/)
+* [Test of English as a Foreign Language (TOEFL)](https://www.ets.org/toefl)
+* [C2 Proficiency, formerly known as Cambridge English: Proficiency (CPE)](https://www.cambridgeenglish.org/exams-and-tests/proficiency/)  
+
+If you haven’t passed an English language test like this, or do not have qualifications in maths or science which are the same standard as grade 4 GCSE, some teacher training providers will let you sit tests in these subjects. It’s best to contact them to ask about their policy before you apply. 
+
+## If you do not have a bachelor's degree
+
+If you do not have a degree, you can apply to study a 3 to 4 year [undergraduate course in England](https://www.ucas.com/postgraduate/teacher-training/applying-teacher-training/find-teacher-training-programmes) which combines a teaching degree with [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts). When you complete your studies, you’ll be able to apply for jobs as a teacher in England.
+
+The [UK Council for International Student Affairs](https://www.ukcisa.org.uk/) has lots of helpful information about studying at an English university.
+
+## How to apply
+
+Have you spoken to one of our advisers to check you have the required qualifications for teacher training in England? They can also help you submit your application. 
```

Comment:
@laurahermionetennant92 Do we stil need this section? We have the section on Get help checking your qualifications and it says if you contact us we can check your qualifications and if you qualify for an adviser. So it feels like this is just repeating it.

Reply:
I think they need to understand what the next step is, otherwise they reach a dead end at the bottom of the page...